testdata/project-v2/_output/config-gen.yaml

```diff
@@ -0,0 +1,392 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: project-v2-system
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
+  creationTimestamp: null
+  name: admirals.crew.testproject.org
+spec:
+  group: crew.testproject.org
+  names:
+    kind: Admiral
+    listKind: AdmiralList
+    plural: admirals
+    singular: admiral
+  scope: Cluster
+  versions:
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          description: Admiral is the Schema for the admirals A
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema '
+              type: string
+            kind:
+              description: 'Kind is a string value representing the '
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: AdmiralSpec defines the desired state of
+              properties:
+                foo:
+                  description: Foo is an example field of Admiral.
+                  type: string
+              type: object
+            status:
+              description: AdmiralStatus defines the observed state
+              type: object
+          type: object
+      served: true
+      storage: true
+      subresources:
+        status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
In the Makefile, the tool has targets to help the users to apply the CRD's, to apply the CRs and etc. 
Could config-gen generate the files in the same directory structure to allow it?